### ISSUE TITLE
Gradient descent, how neural networks learn | Chapter 2, Deep learning

### ISSUE THREAD

**Issue by NPINK001** [NPINK001/Capstone#23]
### Title
Gradient descent, how neural networks learn | Chapter 2, Deep learning
### URL
https://www.youtube.com/watch?v=IHZwWFHWa-w
### Summary 
This video explains how gradient descent works, which is useful in deep learning and other forms of machine learning.
### Key Points 
Neural Networks, Gradient Descent
### Citation
3Blue1Brown. Gradient descent, how neural networks learn | Chapter 2, Deep learning. 2018. https://www.youtube.com/watch?v=IHZwWFHWa-w
